code/datums/diseases/_MobProcs.dm

```diff
@@ -43,7 +43,8 @@
 		SSdisease.active_diseases += DD //Add it to the active diseases list, now that it's actually in a mob and being processed.
 		if(istype(DD, /datum/disease/advance)) //Log the infection
 			var/datum/disease/advance/A = DD
-			SSblackbox.add_details("infections", "[A.name] - Symptoms: [english_list(A.symptoms)]")
+			var/list/L = sortList(A.symptoms)
```

Comment:
You need to create a custom cmp proc in https://github.com/tgstation/tgstation/blob/master/code/__HELPERS/cmp.dm and use that as a second parameter. `sortList(A.symptoms, /proc/cmp_symptom_asc)`